src/components/Components/LinkedSecretsListView/LinkedSecretsListView.tsx

```diff
@@ -0,0 +1,170 @@
+  const applicationBreadcrumbs = useApplicationBreadcrumbs();
+  const [component, componentLoaded, componentError] = useComponent(namespace, componentName);
+  const [activeSortIndex, setActiveSortIndex] = React.useState<number>(SortableHeaders.secretName);
+  const [activeSortDirection, setActiveSortDirection] = React.useState<SortByDirection>(
+    SortByDirection.asc,
+  );
+  const [linkedSecrets, linkedSecretsLoaded, linkedSecretsError] = useLinkedSecrets(
+    namespace,
+    componentName,
+  );
+
+  const LinkedSecretsListHeader = React.useMemo(
+    () =>
+      getListHeader(activeSortIndex, activeSortDirection, (_, index, direction) => {
+        setActiveSortIndex(index);
+        setActiveSortDirection(direction);
+      }),
+    [activeSortDirection, activeSortIndex],
+  );
+
+  const sortedLinkedSecrets = useSortedResources(
+    linkedSecrets,
+    activeSortIndex,
+    activeSortDirection,
+    sortPaths,
+  );
+
+  if (!componentLoaded || !linkedSecretsLoaded) {
+    return (
+      <Bullseye>
+        <Spinner data-test="spinner" />
+      </Bullseye>
+    );
+  }
+
+  if (componentError || linkedSecretsError) {
+    const error = componentError ?? linkedSecretsError;
+    const httpError = HttpError.fromCode(error ? (error as { code: number }).code : 404);
+    return (
+      <ErrorEmptyState
+        data-test="linked-secrets-list-view_error-empty-state"
+        httpError={httpError}
+        title="Unable to load linked secrets"
+        body={httpError?.message.length ? httpError?.message : 'Something went wrong'}
+      />
+    );
+  }
+
+  const EmptyMessage = () => (
+    <FilteredEmptyState
+      onClearFilters={() => {}} // TODO: clear name filter - will be implemented in another ticket
+      data-test="linked-secrets-list-view_filtered-empty-state"
+    />
+  );
+
+  const NoDataEmptyMessage = () => (
+    <AppEmptyState
+      emptyStateImg={emptyStateImgUrl}
+      title="No linked secrets found for this component"
+      data-test="linked-secrets-list-no-data-empty-message"
+    >
+      <EmptyStateBody>To get started, link a new secret.</EmptyStateBody>
+    </AppEmptyState>
+  );
+
+  return (
+    <>
+      <PageLayout
+        title="Manage linked secrets"
+        description={
+          <>
+            You can add new secrets to this component or unlink existing secrets.{' '}
+            <ExternalLink href="">Learn more about linked secrets</ExternalLink>
+          </>
+        }
+        breadcrumbs={[
+          ...applicationBreadcrumbs,
+          {
+            path: COMPONENT_LIST_PATH.createPath({
+              workspaceName: namespace,
+              applicationName,
+            }),
+            name: 'components',
+          },
+          {
+            path: COMPONENT_DETAILS_PATH.createPath({
+              workspaceName: namespace,
+              applicationName,
+              componentName,
+            }),
+            name: component?.spec.componentName || 'Unknown component',
```

Comment:
component.metadata.name is the unique name in the namespace and it is mandatory required.
Based on previous check, i guess it is quite safe to just enjoy component.metadata.name here?

Reply:
good point, using `component.metadata.name` is better here